apps/ledger-live-desktop/src/renderer/components/Select/index.tsx

```diff
@@ -279,7 +275,6 @@ class Select<
         menuPortalTarget={document.body}
         rowHeight={rowHeight}
         onChange={this.handleChange}
-        data-test-id={selectDataTestId}
```

Comment:
this is both unused and also never could have worked. react select don't allow such extra attribute.